core/src/test/java/org/infinispan/statetransfer/ConcurrentStartChanelLookupTest.java

```diff
@@ -103,7 +103,7 @@ public void testConcurrentStart(int eagerManager, int lazyManager) throws Except
 
    private EmbeddedCacheManager createCacheManager(String name1, JChannel ch1) {
       GlobalConfigurationBuilder gcb1 = new GlobalConfigurationBuilder();
-      gcb1.transport().nodeName(ch1.getName()).distributedSyncTimeout(10, SECONDS);
+      gcb1.transport().defaultTransport().nodeName(ch1.getName()).distributedSyncTimeout(10, SECONDS);
```

Comment:
Is this a different bug?

Reply:
No, I just changed `CustomChannelLookup.registerChannel` to not set the transport in the global config because it's not in the name... I guess I could have renamed it to `configureTransportWithChannel()` instead :)